### PR TITLE
Removing the version from the commands related to API Products

### PR DESCRIPTION
**Pull request by wasuradananjith**
## Purpose
Since the UI does not support versioning of API Products, the users may be confused when we are having the version in API Products in the API Controller.

## Goals
Remove the version when exporting an API Product, listing API Products, generate keys for API Products and deleting an API Product.

## Approach
- Removed the version flag defining explicitly by the user.
- Hide the version in helper commands.
- Generated docs with the changes.

## User stories
- Users do not need to worry about the version of an API Product.

## Documentation
Documentation changes will be done.

## Test environment
- Ubuntu 20.04 LTS
- go version go1.14.4 linux/amd64

### CODE REVIEW

File: import-export-cli/docs/apictl_set.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ apictl set --mode default
 ### Options
 
 ```
-      --export-directory string    Path to directory where APIs should be saved (default "/home/wso2user/.wso2apictl/exported")
+      --export-directory string    Path to directory where APIs should be saved (default "/home/wasura/.wso2apictl/exported")
```
Comment (malinthaprasan) [inline import-export-cli/docs/apictl_set.md:31]: shall we revert this change

Comment (wasuradananjith) [reply import-export-cli/docs/apictl_set.md:31]: Sure will do it
